src/powershell-win-core/powershell-win-core.csproj

```diff
@@ -55,5 +53,7 @@
   <PropertyGroup Condition=" '$(Configuration)' == 'CodeCoverage' ">
     <DebugType>full</DebugType>
   </PropertyGroup>
-
+  <PropertyGroup>
+    <RootNamespace>Microsoft.PowerShell</RootNamespace>
```

Comment:
Is the <RootNamespace> necessary to be added?
If so, can you please move it to the `PropertyGroup` section at top of the file?

Reply:
It avoids a warning when opening the solution. I moved it to the top.